pkgs/development/python-modules/python-youtube/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, poetry-core
+, dataclasses-json
+, isodate
+, requests
+, requests-oauthlib
+, pytestCheckHook
+, responses
+, pytest
+, pytest-cov
+,
+}:
+buildPythonPackage rec {
+  pname = "python-youtube";
+  version = "0.9.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "sns-sdks";
+    repo = "python-youtube";
+    rev = "v${version}";
+    hash = "sha256-uimipYgf8nfYd1J/K6CStbzIkQiRSosu7/yOfgXYCks=";
+  };
+
+  postPatch = ''
```

Comment:
Please try to upstream this

Reply:
https://github.com/sns-sdks/python-youtube/pull/159